lib/rets/metadata/lookup_type.rb

```diff
@@ -4,8 +4,8 @@ class LookupType
       attr_reader :long_value, :value
 
       def initialize(lookup_type_fragment)
-        @value       = lookup_type_fragment["Value"]
-        @long_value  = lookup_type_fragment["LongValue"]
+        @value       = lookup_type_fragment["Value"].rstrip
+        @long_value  = lookup_type_fragment["LongValue"].rstrip
```

Comment:
This came from a mis-understanding on my part about some code we have in `estately`'s private repos. We use `rstrip` on values but this happens after they're resolved anyway. I'll switch it to `strip`